bitnami/grafana-loki/templates/query-scheduler/deployment.yaml

```diff
@@ -0,0 +1,165 @@
+{{- if .Values.queryScheduler.enabled }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ template "grafana-loki.query-scheduler.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: grafana-loki
+    app.kubernetes.io/component: query-scheduler
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  minReadySeconds: 10
+  replicas: {{ .Values.queryScheduler.replicaCount }}
+  {{- if .Values.queryScheduler.updateStrategy }}
+  strategy: {{- toYaml .Values.queryScheduler.updateStrategy | nindent 4 }}
+  {{- end }}
+  revisionHistoryLimit: 2
```

Comment:
`revisionHistoryLimit` is not used in other deployments. We should keep consistency between all the deployments in this chart. Could you remove it?